src/components/app.tsx

```diff
@@ -3,21 +3,36 @@ import { connect } from 'react-redux';
 import { AnyAction } from 'redux';
 import { ThunkDispatch } from 'redux-thunk';
 
-import { initWallet } from '../store/actions';
+import { UI_UPDATE_CHECK_INTERVAL } from '../common/constants';
+import { initWallet, updateStore } from '../store/actions';
 
 interface OwnProps {
     children: React.ReactNode;
 }
 
 interface DispatchProps {
     onInitWallet: () => any;
+    onUpdateStore: () => any;
 }
 
 type Props = OwnProps & DispatchProps;
 
 class App extends React.Component<Props> {
+    public state = {
+        isActiveCheckUpdates: false,
+    };
+
     public componentWillMount = () => {
         this.props.onInitWallet();
+        /* Enables realtime updates of the store using pooling */
+        if (!this.state.isActiveCheckUpdates) {
```

Comment:
This looks a little odd to me. First, because using the state of the component for this seems like a stretch. And second because `componentWillMount` should only be called once anyway.

I think a better approach is something like this:

```typescript
if (!this.updateStoreInterval) {
  this.updateStoreInterval = setInterval(...);
}
```

and you can also clear the interval in `componentWillUnmount` (probably not necessary since this is the root, but a good practice anyway). You'll have to declare `updateStoreInterval` as a field of the class; I'm not sure about its type but it should be easy to find out.

Reply:
it's done